test-results.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Test results don't belong in this repo.

please add it to .gitignore so it's excluded